javascript/javascript3/week3/canvas.js

```diff
@@ -0,0 +1,40 @@
+class Circle {
+    constructor(x, y, r, startAngle, endAngle, fillColor) {
+        this.x = x;
+        this.y = y;
+        this.r = r;
+        this.startAngle = startAngle;
+        this.endAngle = endAngle;
+        this.fillColor = fillColor;
+    }
+    draw() {
+        const canvas = document.getElementById('myCanvas');
+        const context = canvas.getContext('2d');
+
+        context.beginPath();
+        context.arc(this.x, this.y, this.r, this.startAngle, this.endAngle);
+        context.fillStyle = this.fillColor;
+        context.fill();
+        context.stroke();
+    }
+}
+
+
+function fillingColor() {
+    return 'rgb(' + Math.floor(Math.random() * 255) + "," + Math.floor(Math.random() * 255) + "," + Math.floor(Math.random() * 255) + ')';
```

Comment:
consider refactoring by introducing new function "getRandomColor()", which will make this statement a lot shorter and cleaner